web/plugs/authorization.ex

```diff
@@ -42,9 +42,9 @@ defmodule Cog.Plug.Authorization do
       raise ":permission key must be a string, but was '#{inspect permission}' instead"
     end
     {_,_} = Permission.split_name(permission) # error if can't be split
-    self_updates = Keyword.get(opts, :allow_self_updates, false)
-    unless self_updates == true || self_updates == false do
-      raise ":allow_self_updates must be a boolean, but was '#{inspect self_updates}' instead"
+    self_updates = Keyword.get(opts, :allow_self_updates, [])
+    unless is_list(self_updates) do
```

Comment:
Might want to expand the check to ensure that all members of the list are atoms.